source/get-started/bootable-usb.rst

```diff
@@ -1,30 +1,19 @@
 .. _bootable-usb:
 
-Create a bootable USB drive
+Create a bootable USB drive using BalenaEtcher
```

Comment:
Change the hash below the header to extend to the last character ("r" in BalenaEtcher)
```suggestion
Create a bootable USB drive using BalenaEtcher
####################################
```